gradle.properties

```diff
@@ -4,10 +4,12 @@ kotlin.stdlib.default.dependency=false
 cliHostVersion= 0.0.1
 
 # PF4J
-pf4jVersion=3.6.0
```

Comment:
We do have a separate Jira for this sort of upgrade: https://r3-cev.atlassian.net/browse/CORE-5398
But I guess we can do it as part of this change.